lib/stats/stats-aggregator-register.c

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright (c) 2021 One Identity
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+#include "stats/stats-aggregator-register.h"
+#include "stats/stats-registry.h"
+#include "stats/stats-query.h"
+#include "cfg.h"
+#include "iv.h"
+#include "timeutils/cache.h"
+#include "mainloop.h"
+
+#define FREQUENCY_OF_UPDATE 60
```

Comment:
[optional] You could always replace this with a constant variable, which has the advantage of having an explicit type.